packages/snippet-bot/src/region-tag-parser.ts

```diff
@@ -97,65 +98,75 @@ export async function parseRegionTagsInPullRequest(
     ) {
       // For the case of renaming the file, we scan the files directly,
       // no need to understand the diffs.
-      const blobBeforeRename = await octokit.repos.getContent({
-        owner: owner,
-        repo: repo,
-        path: file.from,
-        ref: sha,
-      });
-      if (!isFile(blobBeforeRename.data)) {
-        continue;
-      }
-      const fileContentsBeforeRename = Buffer.from(
-        blobBeforeRename.data.content,
-        'base64'
-      ).toString('utf8');
+      try {
+        const blobBeforeRename = await octokit.repos.getContent({
+          owner: owner,
+          repo: repo,
+          path: file.from,
+          ref: sha,
+        });
+        if (!isFile(blobBeforeRename.data)) {
```

Comment:
I don't understand why you'd need to specify this case, if you're simply continuing. Why wouldn't you assert the opposite?